test/26_air_beefmaker_extfn/test.cpp

```diff
@@ -1,203 +1,103 @@
-//===- test.cpp -------------------------------------------------*- C++ -*-===//
+//===- main.cpp -------------------------------------------------*- C++ -*-===//
 //
-// Copyright (C) 2020-2022, Xilinx Inc.
-// Copyright (C) 2022, Advanced Micro Devices, Inc.
+// Copyright (C) 2022, Advanced Micro Devices, Inc. All rights reserved.
 // SPDX-License-Identifier: MIT
 //
 //===----------------------------------------------------------------------===//
 
-#include <cassert>
-#include <cmath>
-#include <cstdio>
-#include <cstring>
-#include <fcntl.h>
-#include <stdlib.h>
-#include <sys/mman.h>
-#include <thread>
-#include <unistd.h>
-
 #include "air_host.h"
-
-#define XAIE_NUM_ROWS            8
-#define XAIE_NUM_COLS           50
-#define XAIE_ADDR_ARRAY_OFF     0x800
-
-#define SHMEM_BASE 0x020100000000LL
-
 #include "test_library.h"
 
-namespace {
-
-XAieGbl_Config *AieConfigPtr;	                          /**< AIE configuration pointer */
-XAieGbl AieInst;	                                      /**< AIE global instance */
-XAieGbl_HwCfg AieConfig;                                /**< AIE HW configuration instance */
-XAieGbl_Tile TileInst[XAIE_NUM_COLS][XAIE_NUM_ROWS+1];  /**< Instantiates AIE array of [XAIE_NUM_COLS] x [XAIE_NUM_ROWS] */
-XAieDma_Tile TileDMAInst[XAIE_NUM_COLS][XAIE_NUM_ROWS+1];
+#include <cassert>
+#include <cstdio>
+#include <dlfcn.h>
+#include <stdlib.h>
+#include <sys/time.h>
+#include <vector>
+
+#define VERBOSE 1
+#define PROFILE 0
+
+/*namespace air::partitions::partition_0 {
+int32_t mlir_aie_read_buffer_beef_0_0(aie_libxaie_ctx_t *ctx, int index);
+} // namespace air::partitions::partition_0*/
```

Comment:
remove this

Reply:
Good catch. Addressed in [f67896e](https://github.com/Xilinx/mlir-air/pull/266/commits/f67896e1d6f1ff6dfccad61e562540cce8725c98).